lib/worldwide/field.rb

```diff
@@ -63,18 +63,42 @@ def base_key(country_key)
 
     def default_lookup(key_suffix, options: {})
       base = base_key("_default")
-      I18n.t("#{base}.#{key_suffix}", default: nil, **options) ||
-        I18n.with_locale(:en) { I18n.t("#{base}.#{key_suffix}", default: nil, **options) }
+
+      message = I18n.t("#{base}.#{key_suffix}", default: nil, **options)
+      return message if message.present?
+
+      message = I18n.t("#{base}.#{key_suffix}_instructional", default: nil, **options)
+      return message if message.present?
+
+      if key_suffix.include?("_instructional")
+        message = I18n.t("#{base}.#{key_suffix.sub("_instructional", "")}", default: nil, **options)
+        return message if message.present?
+      end
+
+      I18n.with_locale(:en) { I18n.t("#{base}.#{key_suffix}", default: nil, **options) }
     end
 
     def lookup(key_suffix, locale:, options: {})
       I18n.with_locale(locale) do
         if @country_code.nil?
-          default_lookup(key_suffix, options: options)
         else
```

Comment:
Instead of an empty `if`, can we return early? 
```rb
return default_lookup(key_suffix, options: options) if @country_code.nil? 

message = I18n.t(...
```